src/helpers/validate.js

```diff
@@ -0,0 +1,12 @@
+const validateFormFields = (formData) => {
+  const fieldArray = Object.keys(formData);
+  const errors = [];
+  for (let i = 0; i < fieldArray.length; i += 1) {
```

Comment:
I would suggest you use `map` instead. [Here](https://medium.com/@JeffLombardJr/understanding-foreach-map-filter-and-find-in-javascript-f91da93b9f2c) are a few reasons why.

Reply:
Fix in `ft-156914066-create-redeem-points-modal` branch

Reply:
Why is this fix in a separate branch? @mkiterian

Reply:
@Chris-Maina fixed the helper when working on his task, the Point Redemptions modal form, so this helper will be replaced one that uses filter instead of the for loop